nifi-nar-bundles/nifi-iceberg-bundle/nifi-iceberg-processors/src/test/java/org/apache/nifi/processors/iceberg/TestIcebergRecordConverter.java

```diff
@@ -221,6 +239,26 @@ private static RecordSchema getPrimitivesSchema() {
         return new SimpleRecordSchema(fields);
     }
 
+    private static RecordSchema getPrimitivesAsCompatiblesSchema() {
+        List<RecordField> fields = new ArrayList<>();
+        fields.add(new RecordField("string", RecordFieldType.STRING.getDataType()));
```

Comment:
I think this field's type should be changed to something else to cover the String conversion.